contrib/time_series_expectations/time_series_expectations/expectations/expect_column_pair_values_to_match_prophet_date_model.py

```diff
@@ -0,0 +1,246 @@
+from typing import Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.execution_engine import (
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnPairMapExpectation
+from great_expectations.expectations.metrics.import_manager import F, sparktypes
+from great_expectations.expectations.metrics.map_metric_provider import (
+    ColumnPairMapMetricProvider,
+    column_pair_condition_partial,
+)
+from time_series_expectations.expectations.prophet_model_deserializer import (
+    ProphetModelDeserializer,
+)
+
+with open(file_relative_path(__file__, "example_prophet_date_model.json")) as f_:
+    model_json = f_.read()
+
+example_data = pd.read_csv(file_relative_path(__file__, "example_data.csv"))
```

Comment:
In this and the other expectations, can we move these inside the class? Just so they aren't run on import, slowing down the library. e.g. put them behind a method like `example_model()` and `example_data()` which are called in the `examples` list instead of using these module level variables.

Alternatively, we can pass them into the function testing the expectations, but I feel like although worthwhile that would be a bigger refactor than is warranted for this expectation package contribution.

Reply:
❤️  I'll do the first version.